bob/inspection.yaml

```
@@ -454,15 +454,11 @@ definitions:
           description: Payment transaction identifier
           type: string
           example: "4B3249E2-EFFD-4C42-B604-D632C58E46FC"
-        rid:
-          description: Registered Application Provider Identifier
+        aid:
+          description: Application Identifier
           type: string
-          pattern: "^[0-9A-Fa-f]{10}$"
+          pattern: "^[0-9A-Fa-f]{10,32}$"
           example: A000000838
```

Update the example to resemble an AID instead of only the RID e.g. 
```suggestion
          example: A0000008381010
```

---

bf0a5b5c1a9cf7adedb9bcc992716eaa20310b7e